EPMEDU-2314: The About Page – Minor Spelling Issues

vadimotion opened the pull request:
## What's new:
The [EPMEDU-2314: Text on the page has several minor spelling issues](https://jira.epam.com/jira/browse/EPMEDU-2314) is fixed:
- Minor spelling issues are fixed on the `About Page`
- `Strings.swift` file generated by the `SwiftGen` tool is included in `.gitignore` file

## Before:
<img width="474" alt="Screenshot 2023-08-23 at 16 23 15" src="https://github.com/AnimealProject/animeal_iOS/assets/36302791/ab767572-bbf7-41d6-93dc-149685f5b7c0">

## After:
<img width="474" alt="Screenshot 2023-08-23 at 16 18 05" src="https://github.com/AnimealProject/animeal_iOS/assets/36302791/af2b6795-64a1-4b7f-886c-286484bfdc76">

vadimotion committed:
EPMEDU-2314: About page: spelling issues are fixed

vadimotion committed:
EPMEDU-2314: .gitignore updated

kishorepran commented on animeal/src/Common/Strings.swift:
Removing unused code is awesome.
Initial look says it might have dependencies at multiple places.
Just to double check if this is and intended change. 
Rest all good.

rinold replied:
Strings.swift is generated file (using SwiftGen) that contains localization enum based on Localizable.strings file. We should not remove it from repo.

kishorepran replied:
@vadimotion  :If possible kindly pull from dev as well. It will help pass the CI/CD checks here.

kishorepran replied:
Thanks @vadimotion: for the update.
@rinold : I have checked that the stings file is reverted. Then entry from gitignore is removed. Kindly review.
Good to go from my side.

kishorepran committed:
Feature/implement script to generate amplify folder (#192)

* add a script to generate the model folder in amplify folder

added a script to generate the model folder in amplify folder
the script was called from the update amplify script

* update .gitignore

Update .gitignore this change in introduced after running the update amplify script
I had other changes as well but did not commit as it might break for people with lower aws amplify versions
will revert this if needed. assuming this dosen't hurt

* Update update_amplify.sh

call the folders check script a bit later in order
apparently the aws tool creates an amplify folder, which later is converted in to amplify-backup folder
the script is tricked where initially it assumes that the folder is there where as in reality the folder is updated during runtime

* update build script and github actions files

remove the amplify codegen models command from the update amplify script command
make the update update script as fail-able
add another step to recover from error

* Update LoginViewModelTests.swift

update build warning for unit test

* Disable test step for now

disabled test step for now as the test is crashing when running test via command line. logs are attached and explained in detail

* Run the recover from error step only if Amplify setup fails

* Delete check_amplify_generated_folders.sh

Remove un used script

rinold reviewed (changes requested):
Please revert the removal of Strings.swift. Also note that this file is added to the .Xcode proj, and removing just the source of it on disk will lead to corrupted file reference in project.

rinold committed:
epmedu-1751: fix resend code for Facebook sign in (#199)

* Fix for resending sms code during Facebook auth

* Minor code fix

* Final fixes for resend code

vadimotion committed:
EPMEDU-2314: after review
